nrfu_tests/test_bgp_peers_state.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2023 Arista Networks, Inc.  All rights reserved.
+# Arista Networks, Inc. Confidential and Proprietary.
+
+"""
+Testcase for verification of BGP IPv4 peers are established.
```

Comment:
Update the same at all other places as well.
```suggestion
Testcase for verification of BGP IPv4 peers state.
```